Firebase/Core/FIRApp.h

```diff
@@ -16,6 +16,11 @@
 
 #import <Foundation/Foundation.h>
 
+#if TARGET_OS_IOS
+// TODO: Remove UIKit import on next breaking change release
```

Comment:
It seems like a good thing for us since we shouldn't unnecessarily drag UIKit in and to let clients know that it will go away.